Code/Geometry/point.h

```diff
@@ -45,9 +45,9 @@ class RDKIT_RDGEOMETRYLIB_EXPORT Point {
 // typedef class Point3D Point;
 class RDKIT_RDGEOMETRYLIB_EXPORT Point3D : public Point {
  public:
-  double x, y, z;
+  double x{0.0}, y{0.0}, z{0.0};
 
-  Point3D() : x(0.0), y(0.0), z(0.0){};
```

Comment:
Swig isn't liking this for the java builds:

```
2020-05-26T06:13:34.7240693Z /home/vsts/work/1/s/Code/Geometry/point.h:48: Error: Syntax error in input(3).
```

I think we may need. swig update.